pkg/session/session.go

```diff
@@ -206,29 +166,17 @@ func Login() {
 		fmt.Println("Already logged in.")
 		return
 	}
-
-	versionReader := bufio.NewReader(os.Stdin)
-	fmt.Printf("Enter kafka version [%s]: ", constants.KafkaVersion)
-
-	version, _ := versionReader.ReadString('\n')
-	version = strings.TrimSpace(version)
-	if version == "" {
-		version = constants.KafkaVersion
-	}
+	version := readUserInput("Enter kafka version [" + constants.KafkaVersion + "]: ")
 	currentSession.version, _ = sarama.ParseKafkaVersion(version)
 
-	reader := bufio.NewReader(os.Stdin)
-
-	fmt.Printf("Enter broker address [%s]: ", constants.KafkaBroker)
-
-	broker, _ := reader.ReadString('\n')
-	broker = strings.TrimSpace(broker)
-	if broker == "" {
-		broker = constants.KafkaBroker
+	currentSession.isSecureKafka = readUserClosedInput("Is your kafka configured with SASL_PLAINTEXT security? (y/n): ")
```

Comment:
the `isSecureKafka` flag should be changed to `isSaslPlaintext`, because there are many ways to secure a kafka cluster